spec/features/index_page_spec.rb

```diff
@@ -24,4 +24,19 @@
 
     expect(page).to have_content(I18n.t('vacancies.index.no_vacancies'))
   end
+
+  context 'with subdomain' do
+    within_subdomain 'galacticempire' do
+      scenario 'lists only the company\'s vacancies' do
```

Comment:
when having a `'` in the String, we prefer to use quotes:

"lists only the company's vacancies"

Reply:
@samuelgrigolato I think you forget about this 😄

Reply:
Indeed! Now it is fixed. Thanks.